specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/examples/CreateOrUpdateASimpleGalleryImageVersion.json

```diff
@@ -16,24 +16,39 @@
               "regionalReplicaCount": 1,
               "encryption": {
                 "osDiskImage": {
-                  "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myDiskEncryptionSet"
```

Comment:
These are edits to the 2019 Gallery version. @kangsun-ctrl  is moving Gallery to the 2020-09-30 version. I suggest syncing with him. https://github.com/Azure/azure-rest-api-specs/pull/12442

Edits to the 2019 version will not by default be included in the new stable 2020-09-30 version unless you make those changes.

Reply:
I merged gallery change into my branch. For crp/diskrp change, does Yemi need to communicate with crp/diskrp pr owner or he can do the work here in this pr?

Reply:
@Sandido CRP will merge my compute.json changes. Do you know if DiskRP is doing any releases for this cycle?